system/lib/emmalloc.c

```diff
@@ -1424,6 +1425,15 @@ size_t emmalloc_compute_free_dynamic_memory_fragmentation_map(size_t freeMemoryS
   return numFreeMemoryRegions;
 }
 
+void emmalloc_dump_free_dynamic_memory_fragmentation_map()
+{
+  size_t freeMemorySizeMap[32];
+  size_t numFreeMemoryRegions = emmalloc_compute_free_dynamic_memory_fragmentation_map(freeMemorySizeMap);
+  printf("numFreeMemoryRegions: %zu\n", numFreeMemoryRegions);
+  for(int i = 0; i < 32; ++i)
```

Comment:
Is there some name/macro we can use instead of the constant 32 here and above?   I guess its already hard coded in the definition of emmalloc_compute_free_dynamic_memory_fragmentation_map?  

It seems like 32 here is the number of bits in a pointer, so maybe we should have it based on sizeof pointer?  Would that also make it work on wasm64 maybe?

Reply:
Expanding this to 64-bit would be ok, although this is for computing fragmentation, so tracking >4GB allocation fragmentation is maybe not that relevant anyways. Anyhow can look into it in a separate pass that expands emmalloc to work on 64-bit in general.